test/e2e/suite/scenario/quickstart.go

```diff
@@ -22,7 +22,10 @@ var _ = Describe("notation quickstart E2E test", Ordered, func() {
 		if err != nil {
 			panic(err)
 		}
-		vhost.SetOption(AuthOption("", ""))
+		vhost.SetOption(
+			AuthOption("", ""),
+			EnableExperimental(),
```

Comment:
Given `notation cert generate-test` is not an experimental feature as discussed in the above comments by others, do should we still need to `EnableExperimental()`?